macos-bundle.sh

```
@@ -0,0 +1,116 @@
+EOT
+
+chmod u+x "$APP/Contents/MacOS/autopsy-launcher"
+
+#
+# Autopsy is copied
+#
+
+echo "Copyng application..."
+SRC="$PWD"  # Change if necessary
+rsync -a --delete   --exclude=".git"  --exclude="target" \
+  "$SRC/${autopsy,bin,etc}" "$APP/Contents/Resources/app/"
```

Shellcheck (0.11.0)

[warning] autopsy is referenced but not assigned.

(SC2154)